graph.go

```diff
@@ -97,9 +102,16 @@ func (g *Graph) Commit() (*QueryResult, error) {
 	return g.Query(q)
 }
 
+// QueryOptionsNew instantiates a new QueryOptions struct.
+func QueryOptionsNew(timeout int) QueryOptions {
```

Comment:
@jeffreylovitz I don't think it's a future robust way, since if you'll have to add more options you'll break the constructor. Perhaps using is a Setter is better.